### PR TITLE
Update app-isolation-release-notes.md

### PR DESCRIPTION
**Pull request by cchavez-msft**

### CODE REVIEW

**Review by alvinashcraft — APPROVED** [review]
This looks great. Thanks for the updates! Do you want to wait for anyone from your team to review, or can I merge this now?

File: desktop-src/SecAuthZ/app-isolation-release-notes.md [hunk 1]
```diff
@@ -9,18 +9,14 @@ ms.localizationpriority: medium
...
-## Windows build 25357 (2023-05-04)
+## Windows build 26100.2161 (2024-10-24)
 
-1. Implicit brokering for the open file dialog and other APIs
-1. Printing
-1. System tray icons
-1. Shell notifications
-1. File system privacy settings to reset file access permissions
-1. Integration with Windows Privacy App Permissions pages
+1. Packages run isolated in supported OSes and fall back to FullTrust in non-supported ones
```
Comment (ddsilva-msft) [inline desktop-src/SecAuthZ/app-isolation-release-notes.md]:
```suggestion
1. Packages run isolated on supported OSes and fall back to FullTrust on non-supported ones
```

**Comment by alvinashcraft** [comment]
@cchavez-msft Is the team happy with this PR now? I can merge when you're ready.

**Review by alvinashcraft — APPROVED** [review]
The latest looks good. I'll go ahead and merge this.

**Pull request merged**